e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -274,5 +276,59 @@ internal static class TestHelper {
             bool isSuccess = json.isSuccess;
             Assert.True(isSuccess);
         }
+
+        /// Wait for all API microservices of IIoT platform to be healthy.
+        /// </summary>
+        /// <param name="config"> IIoT platform configuration </param>
+        /// <param name="ct"> Cancellation token </param>
+        /// <returns></returns>
+        public static async Task WaitForServicesAsync(
+            IIIoTPlatformConfig config,
+            CancellationToken ct
+        ) {
+            const string healthyState = "Healthy";
+
+            var healthRoutes = new string[] {
+                TestConstants.APIRoutes.RegistryHealth,
+                TestConstants.APIRoutes.PublisherHealth,
+                TestConstants.APIRoutes.TwinHealth,
+                TestConstants.APIRoutes.HistoryHealth,
```

Comment:
Please remove the checks, for non-GA services, that will currently not be tested:

* History
* Gateway
* Vault
* Events

Reply:
Done.